controller/lib/core/blower_fsm.cpp

```diff
@@ -67,20 +67,20 @@ class OffFsm {
 // an acceptable waveform, although it remains to be seen.
 class PressureControlFsm {
 public:
-  explicit PressureControlFsm(const VentParams &params)
+  explicit PressureControlFsm(Time now, const VentParams &params)
```

Comment:
Update  comment explaining the "duck-typed interface".

Reply:
Indeed, I missed it.